.github/workflows/codeql.yml

```
@@ -0,0 +1,47 @@
+      matrix:
+        language: [ "csharp" ]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v3
+        with:
+          languages: ${{ matrix.language }}
+          paths: |
```

actionlint (1.7.9)

input "paths" is not defined in action "github/codeql-action/init@v3". available inputs are "analysis-kinds", "build-mode", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "dependency-caching", "external-repository-token", "languages", "matrix", "packs", "quality-queries", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)